src/domain/dataverse/domain.ts

```diff
@@ -110,7 +130,7 @@ export const storeFactory = (
               }
             }
           })),
-        setLanguage: (newLng: string): IO<void> =>
+        setLanguage: (newLng: string): IOType<void> =>
```

Comment:
```suggestion
        setLanguage: (newLng: string): IO.IO<void> =>
```

(if you want to set the return type explicitly)